examples/AspNetCore/README.md

```diff
@@ -0,0 +1,72 @@
+# OpenTelemetry Example Application
+
+This project is an example of ASP.NET Core instrumentation for tracing and
+logging.
+
+The following exporters are configured for viewing the traces:
+
+* Console
+* Jaeger
+* Otlp
+* Zipkin
+
+The following exporters are configured for viewing the metrics:
+
+* Console
+* Otlp
+* Prometheus
+
+## Running the example
+
+Running instances of the following services are required to view the exported
+data:
+
+* Jaeger
+* OTEL Collector
```

Comment:
The official acronym is `OTel`. See: https://github.com/open-telemetry/opentelemetry-specification#acronym

```suggestion
* OTel Collector
```

this comment also applies to other places

Reply:
Thanks for the link, updated!